packages/synapse-react-client/src/components/HasAccess/HasAccessV2.tsx

```diff
@@ -51,18 +62,21 @@ const iconConfiguration: Record<
     color: theme => theme.palette.warning.main,
     tooltipText: 'You do not have download access for this item.',
   },
-
   [RestrictionUiType.AccessibleWithTerms]: {
     icon: 'accessOpen',
     color: theme => theme.palette.success.main,
     tooltipText: 'View Terms',
   },
-
   [RestrictionUiType.Accessible]: {
     icon: 'accessOpen',
     color: theme => theme.palette.success.main,
     tooltipText: '',
   },
+  [RestrictionUiType.AccessibleExternalFileHandle]: {
+    icon: 'linkOff',
+    color: theme => theme.palette.grey[700],
+    tooltipText: 'Access controlled by an external system.',
```

Comment:
Small tweak to make this more generalizable, not sure if the design prescribed this text. Generally, we don't know if there are additional access controls.

```suggestion
    tooltipText: 'Access may be controlled by an external system.',
```

Reply:
The design didn't give me a tooltipText